.gitignore

```diff
@@ -13,6 +13,7 @@ nbactions.xml
 target/
 *.log*
 BlocklyProp.iml
+pom.xml
```

Comment:
We need the pom.xml to be included in the project. Nothing builds correctly without it.